bluebottle/bb_tasks/tests/test_api.py

```diff
@@ -354,6 +354,25 @@ def test_withdraw_task_member_unauthorized(self):
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN,
                          response.data)
 
+    def test_task_member_set_time_spent(self):
+        task = TaskFactory.create(status=Task.TaskStatuses.open,
+                                  project=self.some_project,
+                                  author=self.some_project.owner)
+        task_member = TaskMemberFactory.create(member=self.some_user, task=task)
```

Comment:
@ashutoshb maybe use a different member as the task_member rather than the same as the project owner. It would be good to test that the task_member can't update their own time_spent.